scripts/check_tests_are_captured_in_ci.py

```diff
@@ -0,0 +1,269 @@
+    return webdriverio_config_file_content
+
+
+def get_e2e_test_modules_from_webdriverio_directory() -> List[str]:
+    """Extracts the names of the all test files in core/tests/webdriverio
+    and core/tests/webdriverio_desktop directory.
+
+    Returns:
+        list(str). An alphabetically-sorted list of of the all test files
+        in core/tests/webdriverio and core/tests/webdriverio_desktop directory.
+    """
+    webdriverio_test_suite_modules = []
+    webdriverio_files = os.path.join(
+        os.getcwd(), 'core', 'tests', 'webdriverio')
+    webdriverio_desktop_files = os.path.join(
+        os.getcwd(), 'core', 'tests', 'webdriverio_desktop')
+    for file_name in os.listdir(webdriverio_files):
+        webdriverio_test_suite_modules.append(file_name)
+    for file_name in os.listdir(webdriverio_desktop_files):
+        webdriverio_test_suite_modules.append(file_name)
+
+    return sorted(webdriverio_test_suite_modules)
+
+
+def get_e2e_test_modules_from_webdriverio_config_file() -> List[str]:
+    """Extracts the filenames from the suites object of
+    wdio.conf.js file.
+
+    Returns:
+        list(str). An alphabetically-sorted list of filenames extracted
+        from the wdio.conf.js file.
+    """
+    webdriverio_config_file_content = read_webdriverio_config_file()
+    # The following line extracts suite object from wdio.conf.js.
+    suite_object_string = re.compile(
+        r'suites = {([^}]+)}').findall(webdriverio_config_file_content)[0]
+    test_files_regex = re.compile(r'/([a-zA-Z]*.js)')
+    e2e_test_modules = test_files_regex.findall(suite_object_string)
+    return sorted(e2e_test_modules)
+
+
+def get_e2e_test_suites_from_webdriverio_config_file() -> List[TestSuiteDict]:
+    """Extracts the test suites from the wdio.conf.js file."""
+    webdriverio_config_file_content = read_webdriverio_config_file()
+    # The following line extracts suite object from wdio.conf.js.
+    suite_object_string = re.sub(r'[\n\t\s]*', '', re.compile(
+        r'suites = {([^}]+)}').findall(webdriverio_config_file_content)[0])
+    test_suite_regex = re.compile(r'\b([a-zA-Z_-]*):\[(.*?)\]')
+    e2e_test_suites: List[TestSuiteDict] = []
+    for test_suite_match in test_suite_regex.findall(suite_object_string):
+        test_suite_name = test_suite_match[0]
+        test_suite_modules = [
+            test_suite for test_suite in
+            test_suite_match[1].split(',') if test_suite
+        ]
+        test_suite_modules = [
+            module.strip()[1:-1].replace('./', '')
+            for module in test_suite_modules
+        ]
+        for test_suite_module in test_suite_modules:
+            if test_suite_name in E2E_TEST_SUITES_THAT_ARE_NOT_RUN_IN_CI:
+                continue
+            e2e_test_suites.append(
+                {
+                    'name': test_suite_name,
+                    'module': test_suite_module
+                }
+            )
+    return sorted(e2e_test_suites, key=lambda x: x['name'])
```

Comment:
Why are these sorted? Maybe mention it in the docstring.

Reply:
Sorry, actually doesn't have to be sorted. Only the modules as we are checking equality which I moved to the if statement.